README.md

```diff
@@ -45,7 +45,7 @@ Below is an example of using Woodwork. In this example, a sample dataset of orde
 ```python
 import woodwork as ww
 
-df = ww.demo.load_retail(nrows=100, return_dataframe=True)
+df = ww.demo.load_retail_to_accessor(nrows=100)
```

Comment:
In order for these to be the same, you need to use `init_woodwork=False` in the call to `load_retail_to_accessor` otherwise df will already have Woodwork initialized, which might change the underlying dtypes compared to not initializing.

Reply:
Awesome, thanks!